contracts/protocol/modules/v1/AuctionRebalanceModuleV1.sol

```diff
@@ -796,8 +806,13 @@ contract AuctionRebalanceModuleV1 is ModuleBase, ReentrancyGuard {
             bidInfo.setTotalSupply
         );
 
+        // Settle the auction if the component quantity is max uint256.
         // Ensure that the component quantity in the bid does not exceed the available auction quantity.
-        require(_componentQuantity <= bidInfo.auctionQuantity, "Bid size exceeds auction quantity");
+        if (_componentQuantity == type(uint256).max) {
```

Comment:
This introduces a new capability. To buy the entire position, regardless of the componentQuantity, you need to pass in type(uint256).max. Should update the documentation for this new behaviour.

Reply:
added to natspec here [62422f6](https://github.com/IndexCoop/index-protocol/pull/27/commits/62422f67f6de527c90b8c55a6b1b88adbd3ca753)